fastly/block_fastly_service_backend.go

```diff
@@ -289,6 +295,9 @@ func (h *BackendServiceAttributeHandler) buildCreateBackendInput(service string,
 	// WARNING: The following fields shouldn't have an empty string passed.
 	// As it will cause the Fastly API to return an error.
 	// This is because go-fastly v7+ will not 'omitempty' due to pointer type.
+	if resource["keepalive_time"].(int) > 0 {
+		opts.KeepAliveTime = gofastly.Int(resource["keepalive_time"].(int))
+	}
```

Comment:
I'd actually move this conditional above the WARNING because although it's doing something similar, it's not for the same reason as the following attributes.